<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="USER-20150602CO" name="boc.api.test.Profits_T" tests="3" failures="0" timestamp="6 Jul 2016 03:07:21 GMT" time="180.704" errors="3">
  <testcase name="profits" time="60.206" classname="boc.api.test.Profits_T">
    <error type="java.lang.Throwable">
      <![CDATA[java.lang.Throwable
	at message : expected [true] but found [false] in method : .profits(Profits_T:29)
	at boc.api.test.Profits_T.profits(Profits_T.java:29)
]]>
    </error>
  </testcase> <!-- profits -->
  <testcase name="profits" time="60.210" classname="boc.api.test.Profits_T">
    <error type="java.lang.Throwable">
      <![CDATA[java.lang.Throwable
	at message : expected [true] but found [false] in method : .profits(Profits_T:29)
	at boc.api.test.Profits_T.profits(Profits_T.java:29)
]]>
    </error>
  </testcase> <!-- profits -->
  <testcase name="profits" time="60.288" classname="boc.api.test.Profits_T">
    <error type="java.lang.Throwable">
      <![CDATA[java.lang.Throwable
	at message : expected [true] but found [false] in method : .profits(Profits_T:29)
	at boc.api.test.Profits_T.profits(Profits_T.java:29)
]]>
    </error>
  </testcase> <!-- profits -->
</testsuite> <!-- boc.api.test.Profits_T -->
